documentation/doc.yaml

```diff
@@ -920,6 +920,25 @@ annotations:
     - service
     version_min: "1.4"
     example: ['server-ssl: "true"']
+  - title: enable-grpc
+    type: bool
+    group: enable-grpc
+    dependencies: ""
+    default: "false"
+    description:
+      - Enables GRPC to pods.
+    tip:
+      - Enable HTTP/2 support for backend severs.
+      - Enable GRPC proto
+    values:
+      - "true"
+      - "false"
+    applies_to:
+      - configmap
+      - ingress
+      - service
+    version_min: "1.4"
```

Comment:
We're backporting only bugs, not features: if PR looks good we can easily plan for future releases.

Reply:
or to be more precise this needs 
version_min: "1.5"